src/frontend/staff-portal/src/app/services/date.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'dateSuffix' })
+export class DateSuffixPipe implements PipeTransform {
+    transform(value: string): string {
+
+    let suffix = 'th',
+        day = value;
+
+        if (day === '1' || day === '21' || day === '31') {
+            suffix = 'st'
+        } else if (day === '2' || day === '22') {
+            suffix = 'nd';
+        } else if (day === '3' || day === '23') {
+           suffix = 'rd';
+        }
+
+        return suffix;
+
+    }
+}
```

Comment:
we always get these errors about no blank line, try to clean that up. I know it is annoying.